src/main/kotlin/com/cognifide/gradle/aem/environment/hosts/HostsOptions.kt

```diff
@@ -0,0 +1,33 @@
+package com.cognifide.gradle.aem.environment.hosts
+
+import com.cognifide.gradle.aem.environment.EnvironmentException
+import org.apache.commons.lang3.SystemUtils
+
+class HostsOptions {
+    var list = listOf(
+        Host("127.0.0.1", "example.com"),
```

Comment:
this could not be here; already there is no direct binding gap <-> gradle-aem-multi/gradle-aem-single and there should not be any